### ISSUE TITLE
Presence subscription

### ISSUE THREAD

**Issue by grIskra** [Rhymen/go-whatsapp#228]
Does Presence subscription work for any contact, even if one never had a chat with the contact?

**Comment by baldoius** [comment]
No. You need to have had contact or their "last online"-privacy setting set to "Anybody"
Corrrect me anyone if I'm wrong.

**Comment by webqart** [comment]
Works write me in telegram: peppa100

**Comment by Kml55** [comment]
Can you make this? Is Presence subscription working for any contact without sending message or without opening chat screen?
@grIskra @baldoius @webqart

**Comment by baldoius** [comment]
@Kml55 Presence subscription works for any contact without sending message or without opening chat screen in case their last online privacy setting is set to "Anybody".
